src/business/ControllerInterface.java

```diff
@@ -2,13 +2,10 @@
 
 import java.util.List;
 

```

Comment:
We may need them in the future. But it's ok we can define them when we actually implement them.